src/components/author/author.stories.mdx

```diff
@@ -41,7 +42,30 @@ const megan = {
 Displays an author, or set of authors, for article bylines.
 
 <Canvas>
-  <Story name="Default" args={{}}>
+  <Story
+    name="Default"
+    parameters={{
+      docs: {
+        source: {
+          code: makeTwigInclude('@cloudfour/components/author/author.twig', {
+            authors: [
+              {
+                name: 'Megan Notarte',
```

Comment:
@gerardo-rodriguez I don't think this code example would work, because each of the properties are supposed to be a function, so when twig tries to call author.name(), it can't call a string

Reply:
I wonder if there is a way to specify an inline function in twig syntax

Reply:
Oh...I didn't catch that was calling a function, interesting! I hadn't noticed this pattern in other components. What's the benefit of setting it up as a function versus just passing a value directly?

Reply:
Tyler suggested being able to pass Timber user objects directly, and [most of the properties on those objects are methods](https://timber.github.io/docs/reference/timber-user/#overview)

Reply:
Ah, thank you, @calebeby! TIL 🎓 

> I wonder if there is a way to specify an inline function in twig syntax

🤔

Reply:
We could also change the author component to allow `string | () => string`

Reply:
> We could also change the author component to allow `string | () => string`

Good idea! 💡 

> I wonder if there is a way to specify an inline function in twig syntax

I can't seem to figure this out. 

I'm now not sure how to proceed. If we do decide to change the template logic, that effort seems out of scope for this PR where the intent was only to update the code example.

Hmm... 🤔 

I'll leave this PR open a bit longer but if no one else has any other suggestions for how to approach this component, I'll close the PR. I'll then leave a comment in https://github.com/cloudfour/cloudfour.com-patterns/issues/1289 pointing out this component needs a second look.

Reply:
@cloudfour/dev Any other ideas for how to handle this? If not, I'll close this PR and leave this story as-is.

Reply:
I have no ideas other than to make the story accept strings. But I don't know how to check `typeof` to see if a value is a string or function in twig, so I don't know if that is possible

Reply:
Sorry I'm late to the party here.

In Twig, if you output `{{something}}` and `something` is a property, it will output the property. But if `something` is a function, it will call the function.

This behavior is described under the "Implementation" heading in [this section of the Twig docs](https://twig.symfony.com/doc/3.x/templates.html#variables):

> For convenience's sake foo.bar does the following things on the PHP layer:
> - check if foo is an array and bar a valid element;
> - if not, and if foo is an object, check that bar is a valid property;
> - if not, and if foo is an object, check that bar is a valid method (even if bar is the constructor - use __construct() instead);
> - if not, and if foo is an object, check that getBar is a valid method;
> - if not, and if foo is an object, check that isBar is a valid method;
> - if not, and if foo is an object, check that hasBar is a valid method;
> - if not, return a null value.

This is why you'll see in Timber examples `author.name` referenced and not `author.name()` even though `name` is a method: https://github.com/timber/starter-theme/blob/14605ce1782b7645e6cc5be6df69e0cab30fc480/templates/single.twig#L11

Reply:
Thanks, @tylersticka, I appreciate your response!

If I'm understanding correctly, we don't need the [`author.twig` component to call the functions](https://github.com/cloudfour/cloudfour.com-patterns/blob/188a4ebdc187df071de499fc65431ede1f8bf9bf/src/components/author/author.twig#L14), then?

It currently uses:
- `author.link()`
- `author.name()`
- `author.avatar()`

If so, I can update the template in this PR (plus the code example if needed). Updating the template is _technically_ tangentially related but I'd rather just make the update and take care of it now. 🙂

Reply:
> If I'm understanding correctly, we don't need the [`author.twig` component to call the functions](https://github.com/cloudfour/cloudfour.com-patterns/blob/188a4ebdc187df071de499fc65431ede1f8bf9bf/src/components/author/author.twig#L14), then?

Correct. In fact, dropping the parentheses would probably make these templates more flexible.